fixed category in main paper (#72)
MrLongbottom/P10-Article: sections/Dataset.tex

% stats
This field is fully observed within the dataset and there are originally $58$ different categories in the dataset.
However, while most of these categories cover a significant number of documents, there are some categories which are only used by a few documents.

> Filtering away all categories covering less than $140$ documents (approximately $0.01\%$ of the total document set) and combining them into a single new 'misc' category of size $229$.

Reviewer: ```suggestion
After filtering away all categories covering less than $140$ documents (approximately $0.01\%$ of the total document set) and combining them into a single new 'misc' category of size $229$, $34$ categories remain.
```

> The smallest category consist of $188$ documents, and for all categories, the median category size is $3022$.

Reviewer: ```suggestion
The smallest category consists of $188$ documents, and for all categories, the median category size is $3022$.
```

> The categories from the 3rd quartile and up do become much larger compared to the median, and there is an outlier with 20241 documents.

Reviewer: ```suggestion
\autoref{fig:category_box} shows an overview of the size of the categories after filtering.
The categories from the 3rd quartile and up do become much larger compared to the median, and there is an outlier with 20241 documents.
```
Har bare tilføjet sætningen der blev slettet. Tænker det er ok at nævne at figuren findes, selvom at den er i appendix. Ellers skal figuren alligevel nævnes et andet sted.

Filtering away all categories covering less than $140$ documents (approximately $0.01\%$ of the total document set) and combining them into a single new 'misc' category of size $229$.
The smallest category consist of $188$ documents, and for all categories, the median category size is $3022$.
The categories from the 3rd quartile and up do become much larger compared to the median, and there is an outlier with 20241 documents.
These larger categories seem to be about topics that are written about often or categories that cover a specific newspaper.
All of the category labels can be seen in \autoref{tab:category_table} in \autoref{sec:appendix_meta_data}.

\begin{figure}
	\centering
	\includegraphics[width=1 \linewidth]{figures/category_box.png}
	\caption{Boxplot over the size of the categories. Size meaning the amount of documents with the same category.}
	\label{fig:category_box}

> \end{figure}

Reviewer: Skulle den ikke bare i appendix?

Author: Jeg ved ikke hvor jeg skulle ligge den henne

Reviewer 2: jeg syntes godt vi kunne flytte det til appendix

Reviewer: Den kan nok godt bare smides et sted i category metadata afsnittet i appendix. Der er alligevel andre boxplot figurer der.

Reviewer 2: er dette blevet tilføjet til appendix?
@slayeren @Starrock45

Reviewer: Jeg har nu tilføjet den ved siden af category histogrammerne i appendix.

> All of the category labels can be seen in \autoref{tab:category_table} in \autoref{sec:appendix_meta_data} and the statistics in \autoref{tab:statistics_over_metadata} in \autoref{sec:appendix_meta_data}.

Reviewer: ```suggestion
All of the category labels can be seen in \autoref{tab:category_table} and the statistics in \autoref{tab:meta_prepro_stats}.
```
Afsnittet behøves nok ikke nævnes alligevel.
